src/macro.ts

```diff
@@ -131,12 +131,11 @@ export default class Macro extends StrictMacro {
 
     return this.externalIf(
       myPrimestat() === $stat`mysticality`,
-      Macro.tryHaveSkill($skill`Stuffed Mortar Shell`)
-        .externalIf(
-          myClass() === $class`Pastamancer` && have($skill`Fearful Fettucini`),
-          Macro.trySkillRepeat($skill`Fearful Fettucini`)
-        )
-        .trySkillRepeat($skill`Saucegeyser`)
+      Macro.tryHaveSkill($skill`Stuffed Mortar Shell`).externalIf(
```

Comment:
i would take out the shell i think? not sure. one of the nice things about spellslinging is that half your combats you don't get hit at all. forcing one hit a fight may be part of why railo is having combat trouble